tests/env/start-swift-ceph.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
```

Comment:
@DeepDiver1975 any example of vm selections? About the sudo, I tried to match objectstore.